### DISCUSSION TITLE
Films

### DISCUSSION THREAD

**Discussion by FoX-Sempai** [lAnubisl/LostFilmTorrentsFeed#152] [Ideas]
Можно ли добавить все фильмы под один пункт - фильмы?

**Comment by lAnubisl** [comment]
Теоретически можно но мне кажется в этом мало смысла. Кому нужно скачивать все фильмы подряд?

**Reply by FoX-Sempai** [reply]
Мне например. Я просто в полуавтоматическом режиме скачиваю. Добавил себе РСС в ридер, и когда есть возможность что-то скачать, скачиваю торрент файлы. Если вылезет фильм, который я явно не хочу, просто не скачаю из РСС. Если идея сложная в реализации, тогда прошу прощения. Спасибо за понимание и внимание.

**Reply by lAnubisl** [reply]
я подумаю, что можно сделать. Давно назрел вопрос редизайна и "все фильмы" не просто вписать в текущий UI.

**Comment by anarchyst47** [comment]
Спасибо за обновление, осталось апостроф победить)

![apostr](https://github.com/user-attachments/assets/4035a084-eb7f-4619-acc4-e2cec6269fbb)
![apostr2](https://github.com/user-attachments/assets/ded6c5cc-27a7-4f97-ac25-748ac2aa6d78)

**Reply by lAnubisl** [reply]
https://github.com/lAnubisl/LostFilmTorrentsFeed/issues/226
